voluptuous/validators.py

```diff
@@ -933,3 +934,49 @@ def _get_precision_scale(self, number):
             raise Invalid(self.msg or 'Value must be a number enclosed with string')
 
         return (len(decimal_num.as_tuple().digits), -(decimal_num.as_tuple().exponent), decimal_num)
+
+
+class SomeOf(object):
+    """Value must pass at least some validations, determined by the given parameter
+
+    The output of each validator is passed as input to the next.
+
+    >>> validate = Schema(SomeOf(min_valid=2, validators=[Range(1, 5), Any(float, int), 6.6]))
+    >>> validate(6.6)
+    6.6
+    >>> validate(3)
+    3
+    >>> with raises(MultipleInvalid, 'value must be at most 5, not a valid value'):
+    ...     validate(6.2)
+    """
+
+    def __init__(self, min_valid, validators, max_valid=None, **kwargs):
+        self.min_valid = min_valid or 0
```

Comment:
Why `min_valid` is not a default keyword argument while `max_valid` is?

Reply:
Originally I only needed "minimum valid schemas" and just added "max valid" for completeness. So for me, min_valid was always needed. We can put them both as optional, and assert that either of them has a value (otherwise, what's the point of this validator). 

Sounds good?